### PR TITLE
Readme example wording adjustments

### PR DESCRIPTION
**Pull request by branflake2267**
Some adjustments to the example api section.

### CODE REVIEW

File: packages/google_sign_in/README.md [hunk 1]
```diff
@@ -82,9 +82,13 @@ Future<Null> _handleSignIn() async {
 }
 ```
 
-See more on how to wire it up the [example Google sign application here](https://github.com/flutter/plugins/blob/master/packages/google_sign_in/example/lib/main.dart).
+## Example
 
-See [google_sign_in.dart](https://github.com/flutter/plugins/blob/master/packages/google_sign_in/lib/google_sign_in.dart) for more API details.
+Find the example wiring in the [Google sign example application here](https://github.com/flutter/plugins/blob/master/packages/google_sign_in/example/lib/main.dart).
+
+## Api details
```
Comment (mit-mit) [inline packages/google_sign_in/README.md]: Should all-caps `Api` to `API`

File: packages/google_sign_in/README.md [hunk 2]
```diff
@@ -82,9 +82,13 @@ Future<Null> _handleSignIn() async {
 }
 ```
 
-See more on how to wire it up the [example Google sign application here](https://github.com/flutter/plugins/blob/master/packages/google_sign_in/example/lib/main.dart).
+## Example
 
-See [google_sign_in.dart](https://github.com/flutter/plugins/blob/master/packages/google_sign_in/lib/google_sign_in.dart) for more API details.
+Find the example wiring in the [Google sign example application here](https://github.com/flutter/plugins/blob/master/packages/google_sign_in/example/lib/main.dart).
```
Comment (mit-mit) [inline packages/google_sign_in/README.md]: I'd leave out the ` here` part at the end.

Comment (mit-mit) [inline packages/google_sign_in/README.md]: Change `sign example` to `sign-in example`

**Comment by branflake2267** [comment]
I've pushed up the changes.

**Comment by mit-mit** [comment]
Thanks @branflake2267, LGTM!

**Pull request merged**